src/display/annotation_layer.js

```diff
@@ -86,7 +86,8 @@ AnnotationElementFactory.prototype =
         return new StrikeOutAnnotationElement(parameters);
 
       default:
-        throw new Error('Unimplemented annotation type "' + subtype + '"');
+        warn('Unimplemented annotation type "' + parameters.data.subtype + '"');
+        return null;
```

Comment:
Since we already `warn` in `annotation.js` for this case, I think that we can avoid duplicate warnings here.
Would `return AnnotationElement(parameters);` make sense here, instead of returning `null`?